app/views/layouts/application.haml

```diff
@@ -86,7 +86,7 @@
                 %li= link_to "Galleries", user_galleries_path(current_user)
                 %li= link_to 'Favorites', favorites_path
                 %li= link_to "Post", new_post_path
-              = link_to logout_path, method: :delete, id: :"header-logout" do
+              = link_to logout_path, method: :delete, id: :"header-logout", data: { confirm: 'Press OK to confirm logout' do
```

Comment:
(Missing the trailing curly brace)

```suggestion
              = link_to logout_path, method: :delete, id: :"header-logout", data: { confirm: 'Press OK to confirm logout' } do
```

Reply:
I couldn't seem to push commits when I tried this before, otherwise I'd be pushing to the branch and trying to get this merged XD I might have something misconfigured, not sure if I'm meant to be able to edit it directly